core/src/main/scala/kafka/server/DelegationTokenManager.scala

```diff
@@ -165,10 +112,11 @@ object DelegationTokenManager {
 
 class DelegationTokenManager(val config: KafkaConfig,
                              val tokenCache: DelegationTokenCache,
-                             val time: Time,
-                             val zkClient: KafkaZkClient) extends Logging {
+                             val time: Time) extends Logging {
   this.logIdent = s"[Token Manager on Broker ${config.brokerId}]: "
 
+  protected val lock = new Object()
+
```

Comment:
Cab we remove unused DescribeResponseCallback type variable at line no:125

Reply:
Done